src/main/java/com/booleanuk/Poker.java

```diff
@@ -28,4 +122,23 @@ public String extensionThreeMethods() {
         // You will also need to add the relevant tests to Extension3Test.java
         return "Replace this method with your own methods and tests";
     }
+
+    public void initializeValues()
+    {
+        this.letterValues = new HashMap<>();
+
+        this.letterValues.put("2", 2);
+        this.letterValues.put("3", 3);
+        this.letterValues.put("4", 4);
+        this.letterValues.put("5", 5);
+        this.letterValues.put("6", 6);
+        this.letterValues.put("7", 7);
+        this.letterValues.put("8", 8);
+        this.letterValues.put("9", 9);
+        this.letterValues.put("10", 10);
+        this.letterValues.put("J", 11);
+        this.letterValues.put("Q", 12);
+        this.letterValues.put("K", 13);
+        this.letterValues.put("A", 14);
+    }
 }
```

Comment:
As a small pointer, you could look at simplifying the `winningThreeCardHand` method a bit for clarity. This could make your logic easier to follow and maintain. But overall, you're on the right track.